src/integrationTest/java/uk/nhs/hee/tis/trainee/forms/api/LtftResourceIntegrationTest.java

```diff
@@ -40,14 +37,20 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 import static uk.nhs.hee.tis.trainee.forms.TestJwtUtil.FEATURES_LTFT_PROGRAMME_INCLUDED;
 import static uk.nhs.hee.tis.trainee.forms.dto.enumeration.EmailValidityType.VALID;
+import static uk.nhs.hee.tis.trainee.forms.dto.enumeration.LifecycleState.DRAFT;
+import static uk.nhs.hee.tis.trainee.forms.dto.enumeration.LifecycleState.SUBMITTED;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import io.awspring.cloud.sns.core.SnsTemplate;
 import java.net.URI;
-import java.time.Instant;
+import java.time.*;
```

Comment:
No biggie for this PR, but you might want to tweak your IntelliJ Editor->Code Style->Java->Imports setting to not use '*' style imports (or set `Class count to use import with '*'` to a large value :))

Reply:
Oh that is really useful. I wanna cry every time when the IDE auto change the import, and I have to manually add all them back 😿 
Thanks for that!

Reply:
It did it to me as well with the new laptop. I _still_ can't get the function line continuation indents quite right 😂